toolbox/bulk_processing/validators.py

```diff
@@ -176,6 +176,15 @@ def validate(postcode, **_kwargs):
     return validate
 
 
+def alphanumeric_plus_hyphen_field_values():
+    def validate(value, **_kwargs):
+        stripped_field_value = value.replace("-", "")
+        if not stripped_field_value.isalnum():
+            raise Invalid(f'Field "{value}" is non alphanumeric')
```

Comment:
Worth wording this carefully since we're sharing these outputs across teams, maybe something like `f'Value "{value}" contains non allowed characters`.